Tests/Command/DoctrineCommandTest.php

```diff
@@ -88,6 +88,7 @@ private function getContainer() : ContainerBuilder
             'doctrine_migrations.organize_migrations' => Configuration::VERSIONS_ORGANIZATION_BY_YEAR,
             'doctrine_migrations.custom_template' => null,
             'doctrine_migrations.all_or_nothing' => false,
+            'doctrine_migrations.connection' => 'connection',
```

Comment:
Your test should use the new configuration `connection` instead of this parameter.